client/src/templates/Challenges/components/CompletionModal.js

```diff
@@ -87,14 +87,14 @@ export class CompletionModal extends Component {
     if (Object.keys(files).length) {
       const filesForDownload = Object.keys(files)
         .map(key => files[key])
-        .reduce(
-          (allFiles, { path, contents }) => ({
-            ...allFiles,
-            [path]: contents
-          }),
-          {}
-        );
-      const blob = new Blob([JSON.stringify(filesForDownload, null, 2)], {
+        .reduce((allFiles, { path, contents }) => {
+          const beforeText = `** start of ${path} **\n\n`;
+          const afterText = `\n** end of ${path} **\n\n`;
```

Comment:
```suggestion
          const afterText = `\n\n** end of ${path} **\n\n`;
```
Is the idea to produce the following?
```
** start of a/path **

some content

** end of a/path **
```
If so, you'll need an extra newline.